src/transforms.ts

```diff
@@ -137,6 +138,7 @@ export const createDealOrder = (dealOrder: PalletCreditcoinDealOrder): DealOrder
         repaymentTransferId: repaymentTransferId.unwrapOr(undefined)?.toString(),
         lock: lock.unwrapOr(undefined)?.toString(),
         borrower: borrower.toString(),
+        block: block.unwrapOr(undefined)?.toNumber(),
```

Comment:
When I run the example I get: 
<img width="226" alt="image" src="https://user-images.githubusercontent.com/37932802/163654607-e8e40e2f-9d66-490d-9be2-3121f4ca9a42.png">

Reply:
nevermind, i think this must be because I haven't rebuilt creditcoin.

Reply:
I built your branch on creditcoin to run locally and am still getting the same error. does it run locally for you @nathanwhit?

Reply:
i rebased your branch on dev locally, restarted my creditcoin node and now things are working...